lnst/Tests/Iperf.py

```diff
@@ -32,59 +32,47 @@ def run(self):
         server = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE,
                                   stderr=subprocess.PIPE, close_fds=True)
 
-        try:
-            stdout, stderr = server.communicate()
-            stdout = stdout.decode()
-            stderr = stderr.decode()
-        except KeyboardInterrupt:
-            pass
+        stdout, stderr = server.communicate()
+        stdout = stdout.decode().strip()
+        stderr = stderr.decode().strip()
+
+        if server.returncode > 0:
+            msg = f"iperf {self._role} returncode = {server.returncode}"
+            logging.error(msg)
+            logging.error(f"iperf stderr: {stderr}")
+            self._res_data["msg"] = msg
+            self._res_data["stderr"] = stderr
```

Comment:
I think we should log also the stdout. Similar to what is logged when `returncode == 0`.

Reply:
I added `logging.debug()` of `stdout` into every error branch.